src/anniversary/sf33rd/Source/Game/Flash_LP.c

```diff
@@ -0,0 +1,67 @@
+#include "common.h"
+#include "sf33rd/Source/Game/Flash_LP.h"
+#include "sf33rd/Source/Game/workuser.h"
+#include "sf33rd/Source/Game/WORK_SYS.h"
+#include "sf33rd/Source/Game/SysDir.h"
+#include "sf33rd/Source/Game/sc_data.h"
+
+extern const u8 Lamp_Flash_Data[2][2] = {{0x07, 0x6f}, {0x1e, 0x03}};
+
+void Flash_Lamp(void) {
```

Comment:
The preferred way of doing arg-less funcs is to omit void
```suggestion
void Flash_Lamp() {
```